documentation/documentation-style.md

```diff
@@ -0,0 +1,256 @@
+# Documentation Style
+
+The DNSControl documentation uses Markdown and is rendered by [GitBook](https://docs.gitbook.com). This allows us to use some additional render rendering.
+
+{% hint style="info" %}
+You can learn more about Markdown itself by visiting [Common Mark](https://commonmark.org/help/).
+{% endhint %}
+
+## Text formatting
+
+We support all the classic inline Markdown formatting:
+
+| Formatting    | Markdown version  | Result          |
+|---------------|-------------------|-----------------|
+| Bold          | `**bold**`        | **bold**        |
+| Italic        | `_italic_`        | _italic_        |
+| Strikethrough | `~strikethrough~` | ~strikethrough~ |
+
+## Titles
+
+* Heading 1: `# A first-level title`
+* Heading 2: `## A second-level title`
+* Heading 3: `### A third-level title`
+
+## Links
+
+You can insert three different types of links:
+
+1. [Relative links](#relative-links)
+2. [Absolute links](#absolute-links)
+3. [Email address mailto links](#email-address-mailto-links)
+
+### Relative links
+
+Relative links are links created by linking pages that already exist in the GitHub repository.
+
+{% code title="Markdown syntax" %}
+```markdown
+[Documentation Style](documentation-style.md)
+```
+{% endcode %}
+
+### Absolute links
+
+Absolute links are for external links.
+
+{% hint style="info" %}
+External links will always open in a new tab.
+{% endhint %}
+
+{% code title="Markdown syntax" %}
+```markdown
+[GitHub releases](https://github.com/StackExchange/dnscontrol/releases/latest)
+```
+{% endcode %}
+
+### Email address mailto links
+
+Email address `mailto` links are useful when you want click on a link that will open up the default email client, fill in TO with the email address of your link, and allow them to write an email to send out.
+
+{% code title="Markdown syntax" %}
+```markdown
+[example@example.com](mailto:example@example.com)
+```
+{% endcode %}
+
+## Code blocks
+
+``` `⏎` creates a new code block.
```

Comment:
Thanks, after reading back I think it could be even better. I've rewritten the introduction so I hope the placing triple backticks is worded better. See the [comparison](https://github.com/StackExchange/dnscontrol/compare/8220681b79f21e2292e8f6a99e25bf09fd84f56e..c54e8e4151c3bcdf49a5b93cfd8101a0b5309f75).